lib/dl_api_connector/dl_api_connector/api_schema/source_base.py

```diff
@@ -288,3 +288,4 @@ def make_condition(self, data: dict, **kwargs: Any) -> BinaryCondition:
     join_type = ma_fields.Enum(JoinType)
     managed_by = ma_fields.Enum(ManagedBy, allow_none=True, dump_default=ManagedBy.user, load_default=ManagedBy.user)
     virtual = VirtualFlagField(attribute="managed_by", dump_only=True)
+    required = ma_fields.Bool(load_default=False, required=False, dump_default=True)
```

Comment:
why different load and dump defaults?

Reply:
it's a mistake. Fixed, now both default is `False`